config/crds/migration_v1alpha1_migplan.yaml

```diff
@@ -96,6 +96,22 @@ spec:
             observedGeneration:
               format: int64
               type: integer
+            unsupportedNamespaces:
+              items:
+                properties:
+                  name:
+                    type: string
+                  unsupportedResources:
+                    items:
+                      type: string
+                    type: array
+                required:
+                - name
+                - unsupportedResources
+                type: object
+              type: array
+          required:
+          - observedGeneration
```

Comment:
This change is reverting https://github.com/konveyor/mig-controller/pull/422.  The `observedGeneration` needs to be optional.